externals/googletest/src/makeHook.sh

```diff
@@ -1,7 +1,24 @@
 #!/bin/sh
 
-make clean
-make
+cd build 
+make 
```

Comment:
Actually I forgot why we have the `make clean`.  Can you check how it is done in the other externals?  Or was there a particular reason for removal?